python/dgl/graph.py

```diff
@@ -200,6 +200,37 @@ def __init__(self,
         self._apply_node_func = None
         self._apply_edge_func = None
 
+    def switch_mode(self, readonly=True):
```

Comment:
The name `switch_mode` is too generic and doesn't seem to match the docstring.  Either change the name to `switch_readonly`, or the docstring to `Switch mode of the graph` if future toggles are to be included.